docusaurus.config.js

```diff
@@ -132,7 +132,10 @@ module.exports = {
           trackingID: 'G-85D2WJWZNL',
         },
         theme: {
-          customCss: require.resolve('./src/css/custom.css'),
+          customCss: [
+            require.resolve('./node_modules/docslab/lib/main.css'),
```

Comment:
I suspect there is a better way to include the docslab CSS, but I did not find any guidance in the docusaurus [documentation about styling](https://docusaurus.io/docs/styling-layout#global-styles).

Reply:
I think this is fine. The other thought I'd have would be embedding the styling in the component or doing an import within the component to this widget.

Reply:
I changed this to follow your proposal of [import within the component](https://github.com/rerobots/docslab/blob/b7f3a7533eee96a9bf542d293df904842dc28c55/integrations/docusaurus-theme/src/theme/DocslabCodeBlock/index.tsx#L4).

I confirmed in the Netlify preview that the CSS style is expressed.

Reply:
Awesome. That makes for a cleaner install & lower chance of breaking it in the future.